runhouse/servers/grpc/unary_server.py

```diff
@@ -58,11 +57,8 @@ def InstallPackages(self, request, context):
                 else:
                     raise ValueError(f"package {package} not recognized")
 
-                if (str(pkg)) in self._installed_packages:
```

Comment:
not sure if this is a common path where we could get a lot of time savings, but a bit misleading, if a user uninstalls the library (e.g. system.run(["uninstall xxx"]), future install package calls won't install the library even though it's not actually installed at that moment